{"uuid":"0fef11ac-a5be-4ec5-a9e2-43dea849b6e1","historyId":"fb334033bfa6c45c6391ec3786913d5","fullName":"cucumber.api.testng.AbstractTestNGCucumberTests.feature","labels":[{"name":"package","value":"com.cec.qa.runner.MainRunner"},{"name":"testClass","value":"com.cec.qa.runner.MainRunner"},{"name":"testMethod","value":"feature"},{"name":"parentSuite","value":"Default suite"},{"name":"suite","value":"Default test"},{"name":"subSuite","value":"com.cec.qa.runner.MainRunner"},{"name":"host","value":"GISI-0NVH5P"},{"name":"thread","value":"6464@GISI-0NVH5P.main(1)"}],"links":[],"name":"Runs Cucumber Feature","status":"broken","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"java.lang.AssertionError: no alert found expected [true] but found [false]","trace":"cucumber.runtime.CucumberException: java.lang.AssertionError: no alert found expected [true] but found [false]\r\n\tat cucumber.api.testng.TestNGCucumberRunner.runCucumber(TestNGCucumberRunner.java:69)\r\n\tat cucumber.api.testng.AbstractTestNGCucumberTests.feature(AbstractTestNGCucumberTests.java:21)\r\n\tat sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\r\n\tat sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\r\n\tat sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\r\n\tat java.lang.reflect.Method.invoke(Method.java:498)\r\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)\r\n\tat org.testng.internal.Invoker.invokeMethod(Invoker.java:583)\r\n\tat org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)\r\n\tat org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)\r\n\tat org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)\r\n\tat org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:648)\r\n\tat org.testng.TestRunner.run(TestRunner.java:505)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:455)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:364)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1137)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1049)\r\n\tat org.testng.TestNG.run(TestNG.java:1017)\r\n\tat org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)\r\n\tat org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)\r\n\tat org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)\r\nCaused by: java.lang.AssertionError: no alert found expected [true] but found [false]\r\n\tat org.testng.Assert.fail(Assert.java:96)\r\n\tat org.testng.Assert.failNotEquals(Assert.java:776)\r\n\tat org.testng.Assert.assertEqualsImpl(Assert.java:137)\r\n\tat org.testng.Assert.assertEquals(Assert.java:118)\r\n\tat org.testng.Assert.assertEquals(Assert.java:568)\r\n\tat com.cec.qa.stepdefinations.KidsEventStepDefinitions.message_displays_For_Group_Event(KidsEventStepDefinitions.java:76)\r\n\tat âœ½.Then \"Time slots not available\" Message displays For Group Event(KidsGroupEvent.feature:70)\r\n"},"stage":"finished","start":1573024143940,"stop":1573024265644,"parameters":[{"name":"arg0","value":"KidsGroupEvent validation"}]}